results/sentence-transformers__paraphrase-multilingual-MiniLM-L12-v2/IndonesianIdClickbaitClassification.json

```diff
@@ -1,15 +1,15 @@
 {
   "dataset_revision": "9fa4d0824015fe537ae2c8166781f5c79873da2c",
   "mteb_dataset_name": "IndonesianIdClickbaitClassification",
-  "mteb_version": "1.6.16",
+  "mteb_version": "1.7.14",
   "train": {
-    "accuracy": 0.6775390625,
-    "accuracy_stderr": 0.04996584679505858,
-    "ap": 0.41624199602304834,
-    "ap_stderr": 0.03915041016085825,
-    "evaluation_time": 5.24,
-    "f1": 0.640332696695918,
-    "f1_stderr": 0.046159910394525674,
-    "main_score": 0.640332696695918
+    "accuracy": 0.552197265625,
```

Comment:
This one seems like quite a big change

Reply:
This is because in the initial subsampling the code only selected the first 2048 samples wihtout shuffling the dataset, which was highly imbalanced, see code diff [here](https://github.com/embeddings-benchmark/mteb/pull/527/files#diff-d50d3fede30c0848c0340e5f8a450edc2e2d27aa6d7bf0175b938a50c768bd41)